cli/cmd/proxy/read/test_clusters.json

```diff
@@ -0,0 +1,302 @@
+          "health_status": {
+            "eds_health_status": "HEALTHY"
+          },
+          "weight": 1,
+          "locality": {
+            
+          }
+        }
+      ],
+      "circuit_breakers": {
+        "thresholds": [
+          {
+            "max_connections": 1024,
+            "max_pending_requests": 1024,
+            "max_requests": 1024,
+            "max_retries": 3
+          },
+          {
+            "priority": "HIGH",
+            "max_connections": 1024,
+            "max_pending_requests": 1024,
+            "max_requests": 1024,
+            "max_retries": 3
+          }
+        ]
+      },
+      "observability_name": "self_admin"
+    },
+    {
+      "name": "original-destination",
+      "added_via_api": true,
+      "circuit_breakers": {
+        "thresholds": [
+          {
+            "max_connections": 1024,
+            "max_pending_requests": 1024,
+            "max_requests": 1024,
+            "max_retries": 3
+          },
+          {
+            "priority": "HIGH",
+            "max_connections": 1024,
+            "max_pending_requests": 1024,
+            "max_requests": 1024,
+            "max_retries": 3
+          }
+        ]
+      },
+      "observability_name": "original-destination"
+    },
+    {
+      "name": "static-server.default.dc1.internal.80d0b664-149b-52fa-c6ae-3dad962bdaa8.consul",
+      "added_via_api": true,
+      "host_statuses": [
+        {
+          "address": {
+            "socket_address": {
+              "address": "10.244.0.74",
+              "port_value": 20000
+            }
+          },
+          "stats": [
+            {
+              "name": "cx_connect_fail"
+            },
+            {
+              "name": "cx_total"
+            },
+            {
+              "name": "rq_error"
+            },
+            {
+              "name": "rq_success"
+            },
+            {
+              "name": "rq_timeout"
+            },
+            {
+              "name": "rq_total"
+            },
+            {
+              "type": "GAUGE",
+              "name": "cx_active"
+            },
+            {
+              "type": "GAUGE",
+              "name": "rq_active"
+            }
+          ],
+          "health_status": {
+            "eds_health_status": "HEALTHY"
+          },
+          "weight": 1,
+          "locality": {
+            
+          }
+        }
+      ],
+      "circuit_breakers": {
+        "thresholds": [
+          {
+            "max_connections": 1024,
+            "max_pending_requests": 1024,
+            "max_requests": 1024,
+            "max_retries": 3
+          },
+          {
+            "priority": "HIGH",
+            "max_connections": 1024,
+            "max_pending_requests": 1024,
+            "max_requests": 1024,
+            "max_retries": 3
+          }
+        ]
+      },
+      "observability_name": "static-server.default.dc1.internal.80d0b664-149b-52fa-c6ae-3dad962bdaa8.consul"
```

Comment:
I wonder if we should scrub the UUID out of the string here, and above?

Reply:
If we do, the test will fail (I can just replace it with something else that matches). But this is how it builds the lookup table, using the `"name"`. Do you think the UUID presents a security risk?

Reply:
Iryna says it’s ok.